liquibase-core/src/main/java/liquibase/command/core/InternalSnapshotCommandStep.java

```diff
@@ -112,19 +112,15 @@ public static String printSnapshot(CommandScope commandScope, CommandResults sna
 
         return SnapshotSerializerFactory.getInstance().getSerializer(format.toLowerCase(Locale.US)).serialize((DatabaseSnapshot) snapshotResults.getResult("snapshot"), true);
     }
-//
-//        public void merge(SnapshotCommandResult resultToMerge) {
-//            this.snapshot.merge(resultToMerge.snapshot);
-//        }
-//    }
 
     public static void logUnsupportedDatabase(Database database, Class callingClass) {
         if (LicenseServiceUtils.isProLicenseValid()) {
             if (!(database instanceof MSSQLDatabase
                     || database instanceof OracleDatabase
                     || database instanceof MySQLDatabase
                     || database instanceof DB2Database
-                    || database instanceof PostgresDatabase)) {
+                    || database instanceof PostgresDatabase
+                    || database.getShortName().equalsIgnoreCase("snowflake"))) { //TODO: Update to use SnowflakeDatabase once the liquibase-snowflake module is fully integrated into liquibase-core
```

Comment:
Can the comment here be removed now that we support Snowflake is integrated into liquibase-core?

Reply:
It can be, but we'll deal with that in a separate change since it's unrelated to this one.